### PR TITLE
enable CEE get oauth for idp troubleshooting

### PR DESCRIPTION
**Pull request by dofinn**

### CODE REVIEW

File: deploy/osd-devaccess/02-osd-devaccess-aggregate.ClusterRole.yaml [hunk 1]
```diff
@@ -20,11 +20,12 @@ rules:
       - pods/exec
     verbs:
       - create
-  # devaccess can get and list projects
+  # devaccess can get and list projects and oauths
   - apiGroups:
       - config.openshift.io
     resources:
       - projects
+      - oauths
```
Comment (jewzaam) [inline deploy/osd-devaccess/02-osd-devaccess-aggregate.ClusterRole.yaml:28]: Any reason not to put this in `osd-readers-aggregate` which then rolls up to dedicated-readers, dedicated-admins, and osd-devaccess?

Comment (cblecker) [reply deploy/osd-devaccess/02-osd-devaccess-aggregate.ClusterRole.yaml:28]: Does this provide access to secrets in any namespace?

Comment (jewzaam) [reply deploy/osd-devaccess/02-osd-devaccess-aggregate.ClusterRole.yaml:28]: Access to oauths?  No, oauths CR has references to secrets in openshift-config which are then copied to openshift-authentication.  Access to oauths doesn't imply access to the secrets.

Quickly added the rule to a stage cluster's dedicated-admins-aggregate-cluster clusterrole.  Dedicated-admins can now get oauths.
```
$ oc adm policy who-can get oauths | grep -A100 ^Group
Groups: dedicated-admins
        osd-sre-admins
        osd-sre-cluster-admins
        system:cluster-admins
        system:cluster-readers
        system:masters
        system:serviceaccounts:dedicated-admin
```

But cannot access the secrets:
```
$ oc adm policy who-can get secret -n openshift-config | grep -A100 ^Group
Groups: osd-sre-admins
        osd-sre-cluster-admins
        system:cluster-admins
        system:masters
$ oc adm policy who-can get secret -n openshift-authentication | grep -A100 ^Group
Groups: osd-sre-admins
        osd-sre-cluster-admins
        system:cluster-admins
        system:masters
```

Comment (dofinn) [reply deploy/osd-devaccess/02-osd-devaccess-aggregate.ClusterRole.yaml]: after trawling through `jq` I can not see any reason why not :+1:

**Pull request merged**